src/OrleansSQLUtils/CreateOrleansTables_PostgreSql.sql

```diff
@@ -859,3 +859,213 @@ VALUES
 	WHERE
 		ServiceId = @ServiceId AND @ServiceId IS NOT NULL;
 ');
+
+
+CREATE TABLE storage
```

Comment:
For the sake of consistency with other scripts, using a capital 's' would be nicer here. I.e. define as `Storage`,